src/game/Game.js

```diff
@@ -71,6 +85,35 @@ class Game {
     }
   }
 
+  // Adds local players to the instance.
+  addLocalPlayers() {
+    let { instance } = this;
+    if (instance === undefined) {
+      instance = this.testInstance;
+    }
+    instance.addPlayer({
+      iconID: 1,
+      id: 'local',
+      name: 'local',
+      backgroundColor: '#EE6666',
+      iconColor: '#00ffff',
+    });
+    instance.addPlayer({
+      iconID: 2,
+      id: 'local2',
+      name: 'local2',
+      backgroundColor: '#EEFFF66',
+      iconColor: '#4422ff',
+    });
+    setTimeout(() => {
```

Comment:
This is kind of hackfix, but with the TODO it is reasonable. (localplayers is still just a dev feature)